config/locales/javascript/javascript.en.yml

```diff
@@ -300,5 +300,8 @@ en:
       count:
         one: "1 vote so far"
         other: "<%=count%> votes so far"
+      answer_count:
+        one: "1 vote"
```

Comment:
@jhass told me a few days ago that `zero` was not needed for english and the "count" translation just above doesn't have it and works well, so really no idea where the problem is.